CollAction/Frontend/app/donation/DonationBox.tsx

```diff
@@ -43,25 +44,26 @@ class DonationBox extends React.Component<IDonationBoxProps, IDonationBoxState>
             amount: 0, 
             showError: false, 
             error: "",
-            showDialog: false,
+            showBankDialog: false,
+            isRecurring: false,
             inputUserEmail: "",
             inputUserName: ""
         };
     }
 
-    clearErrors() {
+    clearErrors(): void {
```

Comment:
See my previous comment; typescript is very good at inferring return types; normally you don't have to specify them.

Reply:
I find it clearer to at least specify them on the function level